### PR TITLE
CreateRecipe endpoint - 200 & 400 tested

### PR DESCRIPTION
**Pull request by danrfletcher**

### CODE REVIEW

File: OpenSourceRecipe/Models/Recipe.cs [hunk 1]
```diff
@@ -46,3 +45,35 @@ public class Recipe
...
+public class CreateRecipeDto
+{
+    public string? RecipeTitle { get; set; }
+    public string? TagLine { get; set; }
+    public int Difficulty { get; set; }
+    public int TimeToPrepare { get; set; }
+    public string? RecipeMethod { get; set; }
+    public string? RecipeImg { get; set; }
+    public string? Cuisine { get; set; }
+    public int? ForkedFromId { get; set; } = null;
```
Comment (MichaelAGill) [inline OpenSourceRecipe/Models/Recipe.cs:58]: Tiny thing to note: The `= null` is essentially redundant. Any nullable variable should be null by default when defined.

File: OpenSourceRecipe/Migrations/002_CuisineTable.cs [hunk 2]
```diff
@@ -9,11 +9,14 @@ public override void Up()
     {
         Execute.Sql("CREATE TABLE \"Cuisine\"" +
                     "(" +
-                    "\"CuisineId\" INT NOT NULL PRIMARY KEY," +
+                    "\"CuisineId\" SERIAL PRIMARY KEY," +
```
Comment (ChrisRistoff) [inline OpenSourceRecipe/Migrations/002_CuisineTable.cs]: Revert this, it's not serial by design

File: OpenSourceRecipe/Migrations/003_RecipeTable.cs [hunk 3]
```diff
@@ -9,7 +9,7 @@ public override void Up()
     {
         Execute.Sql("CREATE TABLE \"Recipe\"" +
                     "(" +
-                    "\"RecipeId\" INT NOT NULL PRIMARY KEY," +
+                    "\"RecipeId\" SERIAL PRIMARY KEY," +
```
Comment (ChrisRistoff) [inline OpenSourceRecipe/Migrations/003_RecipeTable.cs]: Revert this and make a new migration below

File: OpenSourceRecipe/Migrations/003_RecipeTable.cs [hunk 4]
```diff
@@ -19,7 +19,7 @@ public override void Up()
                     "\"Cuisine\" TEXT NOT NULL," +
                     "\"RecipeImg\" VARCHAR(255) NOT NULL," +
 
-                    "\"ForkedFrom\" INT DEFAULT NULL," +
+                    "\"ForkedFromId\" INT DEFAULT NULL," +
```
Comment (ChrisRistoff) [inline OpenSourceRecipe/Migrations/003_RecipeTable.cs]: Revert and make a new migration below

File: OpenSourceRecipe/OpenSourceRecipe.csproj [hunk 5]
```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="3.5.8" />
```
Comment (ChrisRistoff) [inline OpenSourceRecipe/OpenSourceRecipe.csproj]: Change version to 13.0.3

File: dotnet-install.sh [hunk 6]
```diff
@@ -0,0 +1,1835 @@
+#!/usr/bin/env bash
```
Comment (ChrisRistoff) [inline dotnet-install.sh]: Remove this file

**Pull request merged**